packages/core-typings/src/Apps.ts

```diff
@@ -59,11 +81,13 @@ export type App = {
 	marketplaceVersion: string;
 	latest: App;
 	status?: AppStatus;
+	subscriptionInfo: AppSubscriptionInfo;
 	licenseValidation?: {
 		errors: { [key: string]: string };
 		warnings: { [key: string]: string };
 	};
 	marketplace: unknown;
```

Comment:
What is this property?

Reply:
Not sure, we'll have to discover and correct it on a future PR.